services/code_hosting/git_hub_service.rb

```diff
@@ -44,7 +44,7 @@ def repos
     # Does the client with the associated credentials have access to the specified repo?
     # @repo [String] Repo URL as string
     def access_to_repo?(repo_url: nil)
-      client.repository?(repo_url.sub("https://github.com/",""))
+      client.repository?(repo_url.sub("https://github.com/", ""))
```

Comment:
I was concerned when this was introduced. This raw substitution instead of a regex, are there any cases where `https://github.com/` is not used? (http?, ssh?, short url?)

Reply:
Right now we're focused on using https github links (most common). In the future, we will 💯 want to support all the things.